### PR TITLE
Updated Build.fx

### PR DESCRIPTION
**Pull request by izavala**

### CODE REVIEW

File: build.fsx [hunk 1]
```diff
@@ -209,6 +209,19 @@ Target "DocFx" (fun _ ->
                     DocFxJson = docsPath @@ "docfx.json" })
 )
 
+//--------------------------------------------------------------------------------
+// Cleanup
+//--------------------------------------------------------------------------------
+
+FinalTarget "KillCreatedProcesses" (fun _ ->
```
Comment (Aaronontheweb) [inline build.fsx]: This needs to get called earlier inside `build.fsx`

Comment (izavala) [reply build.fsx]: I followed the same setup on this build.fx

https://github.com/petabridge/Akka.Persistence.Extras/blob/73fb9c66b36452e1f83d93857ce4b1b7b16b00f2/build.fsx#L282-L294

I also did this on another project, should this be towards the beginning of the file?

Comment (Aaronontheweb) [reply build.fsx]: This is what you're missing: 

https://github.com/petabridge/Akka.Persistence.Extras/blob/73fb9c66b36452e1f83d93857ce4b1b7b16b00f2/build.fsx#L43

Comment (izavala) [reply build.fsx]: Ahh I see, completely missed this. Fixing that now

**Pull request merged**